### DISCUSSION TITLE
Enable ip for accessing external Mysql

### DISCUSSION THREAD

**Discussion by fablabivrea** [Budibase/budibase#12885] [Help]
Hi, i would like to connect a Mysql db from budibase hosted on your server. What ip address do i have to enable on my db server in order to be accessed from budibase server? I think budibase is hosted here 3.248.69.251 ( ec2-3-248-69-251.eu-west-1.compute.amazonaws.com ) but i'm not sure of this. This is the message i receive on error trying to connect from budibase to external db

**Comment by fablabivrea** [comment]
I think it can be 188.114.96.7 but i don't know if it depends from single app

**Reply by ConorWebb96** [reply]
Hey @fablabivrea,

The ips which need whitelisted can be found [here](https://docs.budibase.com/docs/whitelisting).
